src/atoms/Link/Link.native.js

```diff
@@ -0,0 +1,64 @@
+import React, { useContext, useState, useCallback } from 'react';
+import PropTypes from 'prop-types';
+import Text from '../Text';
+import { ThemeContext } from 'styled-components/native';
+import automation from '../../_helpers/automation-attributes';
+import Flex from '../Flex/Flex.native';
+import { TouchableOpacity } from 'react-native';
+
+const linkToTextSize = {
+  small: 'xsmall',
```

Comment:
let's maintain our standards and semantics of small, xsmall etc. Screw design. It's too confusing and annoying to write mappings in each and every file

Reply:
The link design has font sizes of 12,14 & 16 which is xsmall, medium & large. They don't have small in link